sdv/evaluation/single_table.py

```diff
@@ -169,18 +161,9 @@ def get_column_pair_plot(
         else:
             plot_type = plot_type.pop()
 
-    for column_name in column_names:
-        sdtype = metadata.columns.get(column_name)['sdtype']
-        if sdtype == 'datetime':
-            datetime_format = metadata.columns.get(column_name).get('datetime_format')
-            real_data[column_name] = pd.to_datetime(real_data[column_name], format=datetime_format)
-            synthetic_data[column_name] = pd.to_datetime(
-                synthetic_data[column_name], format=datetime_format
-            )
-
-    require_subsample = sample_size and sample_size < min(len(real_data), len(synthetic_data))
```

Comment:
The behavior will change slightly for this: before it only subsampled if sample size was smaller than both. Now it's possible to subsample only one of them,